articles/libraries/auth0-swift/index.md

```diff
@@ -55,33 +59,180 @@ You will need to add an `Auth0.plist` file, containing your Auth0 client id and
+
+To ensure an Open ID Connect compliant responses you must either request an `audience` or enable the **OIDC Conformant** switch in your Auth0 dashboard under `Client / Settings / Advanced OAuth`. You can read more about this [here](/api-auth/intro#how-to-use-the-new-flows).
+
+```swift
+Auth0
+    .webAuth()
+    .audience("https://${account.namespace}/userinfo")
+    .start { result in
+        switch result {
+        case .success(let credentials):
+            print("credentials: \(credentials)")
+        case .failure(let error):
+            print(error)
+        }
+    }
+```
 
-### Login with database connection
+::: note
+Please see [Browser-Based vs. Native Login Flows on Mobile Devices](/tutorials/browser-based-vs-native-experience-on-mobile) for information on choosing between the two types of login flows.
+:::
+
+#### Authenticate with a specific Auth0 connection
+
+The `connection` option allows you to specify a connection that you wish to authenticate with. If no connection is specified here, the browser will show the Hosted Login page, with all of the connections which are enabled for this client.
+
+```swift
+Auth0
+    .webAuth()
+    .connection("facebook")
+    .audience("https://${account.namespace}/userinfo")
+    .start { result in
+        switch result {
+        case .success(let credentials):
+            print("credentials: \(credentials)")
+        case .failure(let error):
+            print(error)
+        }
+    }
+```
+
+#### Authenticate using a specific scope
+
+Using scopes can allow you to return specific claims for specfic fields in your request. Adding parameters to `scope` will allow you to add more scopes. The default scope is `openid`, and you should read our [documentation on scopes](/scopes) for further details about them.
+
+```swift
+Auth0
+    .webAuth()
+    .scope("openid email")
+    .connection("google-oauth2")
+    .audience("https://${account.namespace}/userinfo")
+    .start { result in
+        switch result {
+        case .success(let credentials):
+            print("credentials: \(credentials)")
+        case .failure(let error):
+            print(error)
+        }
+    }
+```
+
+### Getting user information
 
-Logging in with a database connection requires calling `login` with the user's username/email, password, and the name of the connection (such as "Username-Password-Authentication") you wish to authenticate with. The response will be a Credentials object.
+In order to retrieve a user's profile, you call the `userInfo` method and pass it the user's `accessToken`.  Although the call returns a [UserInfo](https://github.com/auth0/Auth0.swift/blob/master/Auth0/UserInfo.swift) instance, this is an basic conformant profile and the only guaranteed claim is the `sub` which contains the user's id, but depending on the requested scope the claims returned may vary.  You can also use the `sub` value to call the [Management API](#Management-API) and return a full user profile.
```

Comment:
Although the call returns a [UserInfo](https://github.com/auth0/Auth0.swift/blob/master/Auth0/UserInfo.swift) instance, this is a basic OIDC conformant profile and the only guaranteed claim is the `sub` which contains the user's id, but depending on the requested scope the claims returned may vary.